docs-markdown/src/test/suite/index-coverage.ts

```diff
@@ -31,9 +34,7 @@ function setupNyc() {
 }
 
 export function run(): Promise<void> {
-
-	const nyc = setupNyc();
-
+	let nyc = setupNyc();
```

Comment:
Settung up nyc here sets up the initial framework of files in the out folder. Without this, line 65 below crashes.

Reply:
@bharney0 - See this to answer your question about needing this line.